react-live.css

```diff
@@ -92,3 +92,9 @@
 .token.deleted {
   color: red;
 }
+
+.tabGuarded{
```

Comment:
I don't think that this CSS is needed. The user agent styles should be good enough right? If it's not showing up, perhaps it's that `overflow: hidden` issue I mentioned earlier...